headers/xdp/libxdp.h

```diff
@@ -75,13 +75,21 @@ const struct btf *xdp_program__btf(struct xdp_program *xdp_prog);
 uint32_t xdp_program__id(const struct xdp_program *xdp_prog);
 int xdp_program__fd(const struct xdp_program *xdp_prog);
 unsigned int xdp_program__run_prio(const struct xdp_program *xdp_prog);
+void __xdp_program__set_run_prio(struct xdp_program *xdp_prog,
+			      unsigned int run_prio);
```

Comment:
Why are you adding these new functions? Just use the ones that are already exported?

Reply:
I tried `xdp_program__set_run_prio`, but it failed. Because the `prog->prog_fd` >= 0. So I remove the check routine 
and adding the `__xdp_program__set_run_prio` function.

Reply:
The checks are there for a reason: setting the values needs to be done before the program is loaded into the kernel, or they won't take effect. So exporting these functions is absolutely not the right way to fix this. At the point you're calling the code `prog->prog_fd` shouldn't be set, so I don't think that's why you're getting that error... My guess would be it's because you're passing in the bit-shifted value instead of just the action value itself?

Reply:
No, run priority is just int value, not bits. I know the reason that `xdp_program__set_run_prio` returns error. I developed these features at commit a7f32c110cd2e2e98a26c64080573e5ade353a06. At that commit, `xdp_program__set_run_prio` used `prog->prog_fd` instead of `prog->prog_fd`. Actually, `prog->prog_fd` == -1, so that check fails.